src/Dfe.ContentSupport.Web/Controllers/ContentController.cs

```diff
@@ -67,12 +48,7 @@ public async Task<IActionResult> Index(string slug, string page = "", bool isPre
             return RedirectToAction(ErrorActionName);
         }
     }
-
-
-    public IActionResult Privacy()
-    {
-        return View();
-    }
+    
 
     [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
     public IActionResult Error()
```

Comment:
Is it worth removing the `Error` and `Privacy` routes as well?

Reply:
The privacy is gone already, I've removed the unused view for it.
The error route allows for the exception to be shown in browser rather than just failing. If I remove it, it will just display the browsers 500 error page. Not sure which way we should go at this point